WordPress/build.gradle

```diff
@@ -493,14 +493,24 @@ dependencies {
     androidTestImplementation libs.squareup.okhttp.mockWebServer
     androidTestImplementation libs.androidx.test.ui.automator
     androidTestImplementation libs.androidx.test.espresso.core, {
+        version {
+            strictly libs.versions.androidxTestEspresso.get()
+        }
         exclude group: 'com.android.support', module: 'support-annotations'
     }
     androidTestImplementation(libs.androidx.test.espresso.contrib) {
+        version {
+            strictly libs.versions.androidxTestEspresso.get()
+        }
         exclude group: 'com.android.support', module: 'appcompat'
         exclude group: 'com.android.support', module: 'support-v4'
         exclude module: 'recyclerview-v7'
     }
-    androidTestImplementation(libs.androidx.test.espresso.accessibility)
+    androidTestImplementation(libs.androidx.test.espresso.accessibility){
```

Comment:
Minor (🔍): Consider adding a space between `)` and `{` at the end of this line. 🙏 

PS: I also noticed this `, {` further above on `androidTestImplementation libs.androidx.test.espresso.core`. Consider also updating this to `androidTestImplementation(libs.androidx.test.espresso.core) {` instead, just like it is already done for all other such configuration. 🙏

Reply:
fixed in 4ebca7d98523f9c84a8bc1859300f15d55f1652f